### PR TITLE
Hosted Login: Explanation of Application External Protocol Values

### PR DESCRIPTION
**Pull request by Will5**
### Your checklist for this pull request

Thank you for your contribution to the `next-dev-docs` repo. 

🚨Please review the [Pull request guidelines](../contributor-guide/contributor-guide.md#pull-request-guidelines) to this repository before submitting this PR.

- [x] Your code builds clean without any errors or warnings
  
  ```bash
  yarn build
  ```

- [x] You are using approved terminology

  Refer to the [style guide](../contributor-guide/style-guide.md) for help.

### Description 

Adds context as to why a developer should choose our recommended configuration(s) when creating an application.

### CODE REVIEW

**Review by Will5 — COMMENTED** [review]
I think this captures the “why” for Protocol, but not for Client Type, PKCE, Token Endpoint Auth Method or Grant Type. Is there anything specific we want to call out, other than higher security option?

File: docs/includes/notes/protocol_oidc.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<details>
```
Comment (Will5) [inline docs/includes/notes/protocol_oidc.mdx:1]: @patriciamcphee, Is there a bast practice around Collapsible Text Blocks? I thought it was cleaner for these particular elements to be collapsible, but if that is a big no, then I can revert these back to the Info Admonition.

Comment (patriciamcphee) [reply docs/includes/notes/protocol_oidc.mdx:1]: For this purpose, they're perfect.

Also, on your include files, make sure it starts with an underscore.

**Comment by patriciamcphee** [comment]
@Will5, can you append an underscore on the include files?  Also, I'd stay away from creating extra folders under /docs/includes because if it applies to any file, it should be at the root. If it's specific to something, like the SDKs, then we would want to keep them separate.

File: docs/integration-guides/next-auth.mdx [hunk 2]
```diff
@@ -69,53 +74,55 @@ You'll overwrite the _wellKnown_, _clientId_, and _clientSecret_ values later in
...
-          idToken: true,
-          checks: ['state', 'pkce'],
-          profile(profile) {
-            return {
-              id: profile.sub,
-              name: profile.sub,
-              email: profile.sub,
-            };
-          },
+  <Tabs groupId="nextjs" queryString>
```
Comment (turnipdabeets) [inline docs/integration-guides/next-auth.mdx:77]: nit: I think @patriciamcphee would like us to indent code blocks when it's under a number like this.

Comment (Will5) [reply docs/integration-guides/next-auth.mdx:77]: I think this is still fine. It is still indented; I just shifted it over one indentation.

**Pull request merged**